falcon/api.py

```diff
@@ -247,7 +247,7 @@ def __call__(self, env, start_response):
         start_response(resp.status, headers)
         return body
 
-    def add_route(self, uri_template, resource):
+    def add_route(self, uri_template, resource, **kwargs):
```

Comment:
Adding `*args` should be OK if we think it will make custom routers easier to use. To be honest, I can't imagine a custom router wanting to add more than one or two additional positional args. Anyway, the mechanics will be invisible to most people, since the docs won’t mention them (and presumably docs for custom routers would only mention the additional params they are using).